org-cyf-itp/content/onboarding/sprints/2/day-plan/index.md

```diff
@@ -0,0 +1,51 @@
++++
+title = 'day-plan'
+layout = 'day-plan'
+emoji= '🧑🏾‍🤝‍🧑🏾'
+menu_level = ['sprint']
+weight = 3
+[[blocks]]
+name="Energiser"
+src="energisers/wemoji"
+[[blocks]]
+name="Morning orientation"
+src="blocks/morning-orientation"
+time=15
+[[blocks]]
+name="Workshop: Asking Questions"
+src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/asking-questions/"
+time="120"
+[[blocks]]
+name="lunch"
+src="blocks/lunch"
+[[blocks]]
+name="Blockers! Getting Unstuck"
+src="module/onboarding/blockers"
+time=45
+[[blocks]]
+name="Study Group"
+src="module/onboarding/development"
+time=45
+[[blocks]]
+name="Afternoon break"
+src="blocks/afternoon-break"
+[[blocks]]
+name="Study Group"
+src="module/onboarding/development"
+time=75
+[[blocks.nested.blocks]]
```

Comment:
Optional structured activity: If trainees / volunteers opt to do this - would these basically run like workshops?

Reply:
Yeah, so, the afternoon activity was not comprehensible to brand new trainees. We didn't have the problem coming from Fundamentals as much, but without them having clear guidance in person people just sort of drifted around. Esp in Zoom School, it has taken a long time for people to grasp how to use the time. 

This is not a huge surprise, obv. So here I'm trying to take some intermediate position -- allow self directed learning but also provide guided activities if those are needed.

Reply:
As per call, of course this depends on classes using the curriculum at all! 🫠